### PR TITLE
[Compound] Improve Bloom

### PR DESCRIPTION
**Pull request by jmartinesp**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

- Simplify bloom effect.
- Display very subtle effect in light mode for avatars with soft colors.

## Motivation and context

Iteration on Bloom.

## Screenshots / GIFs

In the PR.

## Tested devices

- [x] Physical
- [ ] Emulator
- OS version(s): 11

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [x] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR

### CODE REVIEW

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components.event_null_TimelineItemImageViewLight_0_null_0,NEXUS_5,1.0,en].png
Comment (jmartinesp) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components.event_null_TimelineItemImageViewLight_0_null_0,NEXUS_5,1.0,en].png:1]: I have no idea why these images are affected. I mean, they do use blurhash, but their implementation isn't related at all to Bloom's.

Comment (bmarty) [reply tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components.event_null_TimelineItemImageViewLight_0_null_0,NEXUS_5,1.0,en].png:1]: Yes, that's strange... Maybe due to https://github.com/vector-im/element-x-android/pull/1262 ?
But CI was happy...

**Review by bmarty — APPROVED** [review]
LGTM, thanks for iterating on this!

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components.event_null_TimelineItemImageViewLight_0_null_0,NEXUS_5,1.0,en].png
Comment (bmarty) [reply tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components.event_null_TimelineItemImageViewLight_0_null_0,NEXUS_5,1.0,en].png:1]: (other recent screenshot recording are not updating those file, so this is not #1262)

**Pull request merged**